controllers/component_build_controller.go

```diff
@@ -553,18 +553,21 @@ func GeneratePipelineRun(name, namespace, bundle, image string, onPull bool) []b
 		"build.appstudio.redhat.com/target_branch":    "{{target_branch}}",
 	}
 	labels := map[string]string{
-		ComponentNameLabelName: name,
+		ComponentNameLabelName:                  component.Name,
+		"pipelines.appstudio.openshift.io/type": "build",
+		"appstudio.openshift.io/application":    component.Spec.Application,
+		"appstudio.openshift.io/component":      component.Spec.ComponentName,
```

Comment:
@Michkov @mmorhun Any preference for using a string literal or always defining constant?

Reply:
this is good

Reply:
I prefer constants to avoid typos.